### PR TITLE
CAD-1247 update iohk-monitoring fw

### PR DESCRIPTION
**Pull request by CodiePP**
update "iohk-monitoring" fw

### CODE REVIEW

**Comment by CodiePP** [comment]
> +1

we are waiting for approval from Codeowners

File: cardano-config/src/Cardano/Config/Logging.hs [hunk 1]
```diff
@@ -211,8 +211,8 @@ createLoggingFeature ver _ nodecli@NodeCLI{configFile} = do
    liveViewdisablesStdout LiveView logConfig = do -- filter out console scribes
      scribes <- CM.getSetupScribes logConfig
      let newscribes = flip filter scribes $ \case
-                        (ScribeDefinition StdoutSK _ _ _ _) -> False
-                        (ScribeDefinition StderrSK _ _ _ _) -> False
+                        (ScribeDefinition StdoutSK _ _ _ _ _ _) -> False
+                        (ScribeDefinition StderrSK _ _ _ _ _ _) -> False
```
Comment (erikd) [inline cardano-config/src/Cardano/Config/Logging.hs:215]: Better to have:
```
     ScribeDefinition StderrSK {} -> False
```

**Pull request merged**